components/cam/src/physics/crm/crm_physics.F90

```diff
@@ -227,6 +237,23 @@ subroutine crm_physics_init(species_class)
    end if
 #endif
 
+   allocate (cldo_save(pcols,pver), stat=ierror)
```

Comment:
Is it ok that there's no deallocate statement for these module variables?

Reply:
These are "global" variables for the module, and the original declarations never "went away", so I assume that having these variables hang around until the end of the model run is the intent.